### DISCUSSION TITLE
How do you view users authenticated via an Identity Provider or Domain Discovery?

### DISCUSSION THREAD

**Discussion by neverstew** [zitadel/zitadel#6122] [General]
As a manager of an organisation, how do I view users that authenticated via an external identity provider or via domain discovery?

**Comment by mffap** [comment]
You should see the users in the Console: https://zitadel.com/docs/guides/manage/console/users

But you also need to make sure that you have the correct permissions as a manager to view users, check out: https://zitadel.com/docs/guides/manage/console/managers

**Reply by neverstew** [reply]
Perhaps there is a bug then... I don't see any users in the console!

**Reply by mffap** [reply]
Can you describe what you're doing?

**Reply by neverstew** [reply]
Here's a slimmed down example.

Two orgs:

1. Org A
The default org for the instance.
Has one application, using PKCE OIDC configuration.
Registration on.

2. Org B
Has one application, using PKCE.
Registration off.
Domain discovery on.

User 1 registers herself via the Org A web app.
Is a user in Org A.
Is a User Manager in Org B.

User 2 registers herself via the Org B web app.
Is a user in Org A (because it's the default domain and registration is off in Org B).

User 2 is not in either Users or Authorizations on the Org 2 console so User 1 (Org 2 manager) has no way of seeing User 2.

**Reply by mffap** [reply]
> User 2 is not in either Users or Authorizations on the Org 2 console so User 1 (Org 2 manager) has no way of seeing User 2.

What's Org 2? Do you mean Org B?

**Reply by mffap** [reply]
But in general, yes a manager is scoped to one organization. But you can also give manager roles to users in another organization. 

There's a tooltip on the dialog: 

<img width="376" alt="Screenshot 2023-07-04 at 11 33 57" src="https://github.com/zitadel/zitadel/assets/1366906/55f1a3ae-a411-408a-8b65-6a908a9e0d02">

**Reply by neverstew** [reply]
> What's Org 2? Do you mean Org B?

Yes, sorry!

> a manager is scoped to one organization

So are you saying that a user manager in Org B won't be able to see any users that have signed in via domain discovery? Even though they've signed in to an application in their organisation?

This doesn't seem right to me!

**Reply by mffap** [reply]
> So are you saying that a user manager in Org B won't be able to see any users that have signed in via domain discovery? Even though they've signed in to an application in their organisation?
> This doesn't seem right to me!

Not sure that I said that. I understood that both User 1 and User 2 are in Org A, but the user Manager is in Org B. Managers from Org B should not see users from Org A, except they are also granted Manager permissions in Org A (or have IAM level permissions anyways).

**Reply by neverstew** [reply]
Yes both users are registered in Org A, because of the default domain/domain discovery combination. But User 2 is signing in to an application in Org B, so currently, even though access to the application in Org B is granted, a user manager in Org B will never be able to see the users that logged into their service.

**Reply by mffap** [reply]
How do you mean "access to the application in Org B is granted"? To whom did you grant it?

**Reply by neverstew** [reply]
User 2 registers/signs in to the Org B application.

**Comment by mffap** [comment]
That's the same discussion as in https://github.com/zitadel/zitadel/discussions/6074, right?

**Reply by neverstew** [reply]
It covers some of the same questions, yeah.
